<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestQML</string>
  </property>
  <widget class="QGroupBox" name="grpPatientWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>571</width>
     <height>501</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Patient Widget</string>
   </property>
   <widget class="QWebView" name="wvPatient">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>551</width>
      <height>451</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>10</horstretch>
      <verstretch>10</verstretch>
     </sizepolicy>
    </property>
    <property name="url">
     <url>
      <string>http://www.evimed.com/erecruitment/</string>
     </url>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpLogin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Login Data</string>
   </property>
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblCenterName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>center Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDepartmentName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>department Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEmailAddress">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>email Address</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpPatientList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Patient List</string>
   </property>
   <widget class="QComboBox" name="cmbPatientList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpPatientDetails">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>241</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>PatientDetails</string>
   </property>
   <widget class="QTableWidget" name="tblPatientDetails">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpStatus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>821</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="lblStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
  <zorder>grpPatientDetails</zorder>
  <zorder>grpLogin</zorder>
  <zorder>grpPatientWidget</zorder>
  <zorder>wvPatient</zorder>
  <zorder>grpPatientList</zorder>
  <zorder>grpStatus</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
